src/test/java/org/sagebionetworks/bridge/services/ParticipantFileServiceTest.java

```diff
@@ -175,7 +174,7 @@ public void createParticipantFileInvalidFile() {
         service.createParticipantFile("api", "test_user", file);
     }
 
-    @Test(expectedExceptions = EntityAlreadyExistsException.class)
+    @Test
     public void createParticipantFileAlreadyExists() {
```

Comment:
I think this test can be just deleted. Since mocks have no state, the mockDao won't "remember" that a previous file of the same ID has already been created. Since we no longer query the mockDao for the existing file, this test functionally does the same thing as createParticipantFile().